src/sentry/web/frontend/setup_wizard.py

```diff
@@ -60,15 +60,15 @@ def get(self, request: Request, wizard_hash) -> HttpResponse:
             member_set__role__in=[x.id for x in roles.with_scope("org:read")],
             member_set__user_id=request.user.id,
             status=OrganizationStatus.ACTIVE,
-        ).order_by("-date_added")[:50]
+        ).order_by("-date_added")[:100]
 
         filled_projects = []
 
         for org in orgs:
```

Comment:
The problem here is that an increase in the number of orgs will result into more queries down the line. The best way to check if this is a problem is to measure the performance.

Reply:
It's a good point. I can also just leave the org limit at 50. What's more important to me is that we increase the limit in projects. Right now, for example, I have 3 orgs but two of those orgs have more than 50 projects.

Reply:
I lack context to judge here, I would say let's go with this PR and if we see any memory pressure on the db end we can tweak it.

Reply:
What is the `ProjectKey` btw?

We can estimate the number of queries in the worst case to:
- 1 query for all orgs
- 100 queries for all projects of each org
- 100 * 500 queries for project key supposing that each org has 500 projects

This results in 50.000 queries 😅

Reply:
> What is the ProjectKey btw?

It's used to query the DSN which we need in the wizard to set up `Sentry.init` options

> This results in 50.101 queries 😅
Versus 1 + 50 + (50 * 50) = 2551

Yup, that's a big difference 😅 

I just want to unblock the most pressing problem which is missing projects. We can be a little more "sane" with the numbers here and only increase the project limit to say 150. Would result in a worst case of 1 + 50 + (50 * 150) = 7551 queries. Still not great of course but a little less bad than the 100/500 limits 😅